capstone/src/main/java/com/google/sps/data/RestaurantDetailsGetter.java

```diff
@@ -41,15 +57,29 @@ public PlaceDetails request(String placeName) {
     try {
       PlacesSearchResult[] results =
           PlacesApi.findPlaceFromText(
-                  context, placeName, FindPlaceFromTextRequest.InputType.TEXT_QUERY)
+              context, placeName, FindPlaceFromTextRequest.InputType.TEXT_QUERY)
               .await()
               .candidates;
       String placeID = results[0].placeId;
       place = getDetails(placeID);
       
     } catch (Exception e) {
-      e.printStackTrace();
+    //   e.printStackTrace();
```

Comment:
any reason we comment?

Reply:
will uncomment

Reply:
I agree this should be uncommented. Removed this try catch in PR(#66) so no longer need this change.